installer/client/src/components/DatabaseConfigDialog.vue

```diff
@@ -14,8 +14,8 @@
         class="orangehrm-database-config-dialog-content orangehrm-database-config--title"
       >
         The following features and add-ons are not supported in OrangeHRM
```

Comment:
we can remove this now right? we have completed all 4.x features except Marketplace that we are going to discontinue anyway

Reply:
yeah the dialog just shows the marketplace and toggle discontinuation. okay to remove it completely? I feel there might be someone who upgrades from 4.x and wonders where they have gone.

![image](https://github.com/orangehrm/orangehrm/assets/97493184/07b13975-5c0c-4687-a347-bb5461f29243)